code/modules/projectiles/projectile/energy.dm

```diff
@@ -33,7 +33,7 @@
 					H.confused = max(H.confused, flash_strength + 5)
 					H.eye_blind = max(H.eye_blind, flash_strength)
 					H.eye_blurry = max(H.eye_blurry, flash_strength + 5)
-
+					H.adjustHalLoss(22) // Five flashes to stun.  Bit weaker than melee flashes due to being ranged.

```

Comment:
Any chance/logic to have this scale based on flash_strength? There's a shotgun flash round that uses it too... Not really a big deal, though.